NAMESPACE

```diff
@@ -28,6 +28,7 @@ export(oauth_app_from_json)
 export(request_build)
 export(request_develop)
 export(request_make)
+export(response_process)
```

Comment:
I don't love this name but we've already got a very strong `NOUN_VERB()` pattern going here and it feels very right for `credentials_*()`, `request_*()`, and `token_*()`. So it feels like I'm committed now and must see this through.